webapp/app/viewModels/role/roleList.js

```diff
@@ -1,20 +1,25 @@
 require('services/roleService');
 
 define(['services/roleService'], function (roleService) {
+    
   var vm = {
     getView: function () {
       var view = require('./roleList.html');
       return $.parseHTML(view)[0];
     },
-    roles: ko.observableArray([])   
+    roles: ko.observableArray([]),
+    roleResponse: ko.observable(''),
+    remove: function (role) {       
+        roleService.roleDelete(role.Id).then(function (response) {
+            vm.roleResponse(response.ProcessJsonRequestsPostResult[0].Value);
```

Comment:
What do you think about showing a nice message instead of the raw response? Something like "The role {roleName}has been successfully removed"